Dockerfile

```diff
@@ -444,7 +444,7 @@ RUN mkdir -p /opt/php/bin \
 # nuget-inspector
 ENV NUGET_INSPECTOR_HOME=/opt/nuget-inspector/bin
 ENV PATH=$PATH:$NUGET_INSPECTOR_HOME
-ARG NUGET_INSPECTOR_VERSION=0.9.6
+ARG NUGET_INSPECTOR_VERSION=0.9.7
```

Comment:
Please squash all nuget-inspector version bumps into the initial commit that introduces nuget-inspector. As we're reviewing (bigger) PRs commit-by-commit, this otherwise wastes a reviewer's time by first reviewing intermediate steps that will anyway be corrected later.

Reply:
All is squashed in the latest push.